docs/reference/hooks.rst

```diff
@@ -558,6 +558,30 @@ Page explorer
   The ``priority`` argument controls the order the buttons are displayed in. Buttons are ordered from low to high priority, so a button with ``priority=10`` will be displayed before a button with ``priority=20``.
 
 
+.. register_page_listing_more_buttons:
+
+``register_page_listing_more_buttons``
+~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~
```

Comment:
This generates a `WARNING: Title underline too short` warning - please can you fix?